### PR TITLE
Update deployment-azure-portal.md 

### PR DESCRIPTION
**Pull request by dasmikko**
I've added more information to help with setting up Nuxt on a Azure Web service

### CODE REVIEW

File: en/faq/deployment-azure-portal.md [hunk 1]
```diff
@@ -3,6 +3,51 @@ title: How to deploy on Azure Portal?
...
 ---
 
+## Requirements
+- It is required that you select a backend when setting up the project. Even if you don't need it, or else the site wont start up.
+- The server is running Node 8 or greater
+
+## What if I already have a project without an backend?
+No worries. It is easy to add an express server to an existing project.
+
+Download the index.js file from this gist: https://gist.github.com/dasmikko/bef743ad4400f8b1c1c8c6e494f081d6 and place it inside a folder called `server` in the root of your project.
```
Comment (manniL) [inline en/faq/deployment-azure-portal.md]:
```suggestion
Use the following index.js file and place it inside a folder called `server` in the root of your project.
```

Could you add the index.js directly in the MD file?

```js
/**
* Place this file inside the "/server" folder in the root project
*/
const express = require('express')
const consola = require('consola')
const { Nuxt, Builder } = require('nuxt')
const app = express()

// Import and Set Nuxt.js options
const config = require('../nuxt.config.js')
config.dev = process.env.NODE_ENV !== 'production'

async function start () {
  // Init Nuxt.js
  const nuxt = new Nuxt(config)

  const { host, port } = nuxt.options.server

  // Build only in dev mode
  if (config.dev) {
    const builder = new Builder(nuxt)
    await builder.build()
  } else {
    await nuxt.ready()
  }

  // Give nuxt middleware to express
  app.use(nuxt.render)

  // Listen the server
  app.listen(port, host)
  consola.ready({
    message: `Server listening on http://${host}:${port}`,
    badge: true
  })
}
start()
```

File: en/faq/deployment-azure-portal.md [hunk 2]
```diff
@@ -3,6 +3,51 @@ title: How to deploy on Azure Portal?
...
+
+Then edit your nuxt.config.js, like so:
```
Comment (manniL) [inline en/faq/deployment-azure-portal.md]:
```suggestion
Then edit your nuxt.config.js:
```

File: en/faq/deployment-azure-portal.md [hunk 3]
```diff
@@ -3,6 +3,51 @@ title: How to deploy on Azure Portal?
 description: How to deploy a Nuxt.js application on Azure Portal?
 ---
 
+## Requirements
+- It is required that you select a backend when setting up the project. Even if you don't need it, or else the site wont start up.
```
Comment (manniL) [inline en/faq/deployment-azure-portal.md]:
```suggestion
- It is required that you select a backend when setting up the project. Even if you don't need it, or else the site won't start up.
```

Comment (manniL) [inline en/faq/deployment-azure-portal.md]: Is this really the case?s 🤔

File: en/faq/deployment-azure-portal.md (hunk 1 above)
Comment (dasmikko) [reply en/faq/deployment-azure-portal.md]: It's inside it now.

File: en/faq/deployment-azure-portal.md (hunk 3 above)
Comment (dasmikko) [reply en/faq/deployment-azure-portal.md]: Yes, from my testing and experience. I was unable to fire up the default nuxt server on on my Azure Web Service.

**Comment by dasmikko** [comment]
Any other changes? :)

**Comment by husayt** [comment]
@manniL  can this be merged or anything is outstanding?

**Comment by dasmikko** [comment]
Nothing is outstanding, go ahead and merge. 👍

**Pull request merged**